src/store/store.ts

```diff
@@ -186,6 +190,24 @@ const reducer = createReducer(initialState, (builder) => {
         userToEdit.skills = skills;
       }
     })
+    .addCase(CREATE_STATUS, (state: RootState, action: CreateStatus) => {
+      const newStatus = action.payload;
+      state.statuses.push(newStatus);
+    })
+    .addCase(DELETE_STATUS, (state: RootState, action: DeleteStatus) => {
+      const statusToDeleteIndex = action.payload;
```

Comment:
I think if the status had an id it will be more suitable for this job than index.

Reply:
done